src/services/etherspot.js

```diff
@@ -583,20 +567,31 @@ export class EtherspotService {
       });
   }
 
-  async getSupportedAssets(): Promise<?(Asset[])> {
+  async getChainSupportedAssets(chain: Chain): Promise<?(Asset[])> {
```

Comment:
```suggestion
  async getSupportedAssets(chain: Chain): Promise<?(Asset[])> {
```

Reply:
On one hand it makes sense, on other it doesn't, but the fact that it accepts only param, which is `chain` already, gives me motion to follow this 👍